doc improvement: Doc fixes for 2.4 (#11171)
nrfconnect/sdk-nrf: doc/nrf/app_dev/bootloaders_and_dfu/bootloader_adding.rst

The S1 variant is built as a separate child image called ``s1_image``.
For this reason, any modifications to the configuration of the S1 variant must be done to the ``s1_image`` child image.
By default, this child image is an exact duplicate of the original image, with the exception of its placement in memory.

> The only configuration option that must be modified is the version set in :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION`.

Reviewer: ```suggestion
You only have to modify the version set in the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option.
```
